LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019- Elastic N.V.
```

Comment:
I assume this is identical to some other Repos?

Reply:
Yes, I added the `-` however. If that's a problem glad to change in a followup.